backend/src/main/java/gov/cdc/usds/simplereport/service/AzureStorageQueueFhirReportingService.java

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.usds.simplereport.service;
+
+import static gov.cdc.usds.simplereport.api.converter.FhirConverter.createFhirBundle;
+
+import ca.uhn.fhir.context.FhirContext;
+import com.azure.storage.queue.QueueAsyncClient;
+import gov.cdc.usds.simplereport.db.model.TestEvent;
+import java.util.concurrent.CompletableFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class AzureStorageQueueFhirReportingService implements TestEventReportingService {
+  private static final Logger log =
```

Comment:
Is there any reason not to use the `@Slf4j` annotation? It's mostly what we're using elsewhere.

Reply:
Same goes for `@AllArgsConstructor`.